frontend/src/component/Account/UserSpacesConfig.jsx

```diff
@@ -0,0 +1,228 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { translate } from 'react-i18next'
+
+import {
+  BtnSwitch,
+  IconButton,
+  ConfirmPopup,
+  ROLE_LIST,
+  TracimComponent,
+  TLM_ENTITY_TYPE as TLM_ET,
+  TLM_CORE_EVENT_TYPE as TLM_CET
+} from 'tracim_frontend_lib'
+
+import { newFlashMessage } from '../../action-creator.sync.js'
+import { deleteWorkspaceMember, getUserWorkspaceList, getWorkspaceMemberList } from '../../action-creator.async.js'
+
+export class UserSpacesConfig extends React.Component {
+  constructor (props) {
+    super()
```

Comment:
```suggestion
    super(props)
```

Reply:
This was discouraged by VSCodium, but it seems to be an issue with VSCodium: https://stackoverflow.com/questions/63883401/react-native-context-and-deprecated-super-usage

Some interesting read on the subject: https://overreacted.io/why-do-we-write-super-props/